### PR TITLE
MPH-160

### PR DESCRIPTION
**Pull request by hboutemy**
https://issues.apache.org/jira/browse/MPH-160

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/help/EffectivePomMojo.java [hunk 1]
```diff
@@ -82,6 +91,14 @@
...
     private String artifact;
 
+    /**
+     * Output POM input location as comments.
+     * 
+     * @since 3.2.0
+     */
+    @Parameter( property = "verbose", defaultValue = "false" )
+    private boolean verbose = false;
+
```
Comment (eolivelli) [inline src/main/java/org/apache/maven/plugins/help/EffectivePomMojo.java]: I think 'verbose' it too generic.
What about 'outputLocations' ?

Comment (hboutemy) [reply src/main/java/org/apache/maven/plugins/help/EffectivePomMojo.java]: we need something that is user friendly

Comment (eolivelli) [reply src/main/java/org/apache/maven/plugins/help/EffectivePomMojo.java]: Ok

**Comment by jira-importer** [comment]
Resolve #163

**Comment by jira-importer** [comment]
Resolve #163

**Pull request closed without merging**